### PR TITLE
Update changelog for 0.8.1 release

### PR DESCRIPTION
**Pull request by tjtelan**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,9 +1,16 @@
...
+* Add GH Release step to release, update changelog ([#1024](https://github.com/infinyon/fluvio/pull/1024))
+* Fix Docker image publish flow ([#1025](https://github.com/infinyon/fluvio/pull/1025))
+* Use file name for the external commands (fixes #889) ([#1008](https://github.com/infinyon/fluvio/pull/1008))
+* Disable test that is flaky on CI ([#1018](https://github.com/infinyon/fluvio/pull/1018))
+* Fix Fluvio log directory on K8 ([#1043](https://github.com/infinyon/fluvio/pull/1043))
+* Add RecordKey API for sending records without keys ([#985](https://github.com/infinyon/fluvio/pull/985))
+* Make Fluvio Client compatitble with WASM ([#1042](https://github.com/infinyon/fluvio/pull/1042))
+* Update Replication logic for SPU ([#1011](https://github.com/infinyon/fluvio/pull/1011))
+* Remove obsolete api from fluvio socket ([#1045](https://github.com/infinyon/fluvio/pull/1045))
+* Remove cargo patch ([#1046](https://github.com/infinyon/fluvio/pull/1046))
```
Comment (sehz) [inline CHANGELOG.md:12]: Should remove 1045, 1046 since these are not functionality changes

**Review by sehz — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (sehz) [reply CHANGELOG.md:12]: and just tempo changes

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,9 +1,16 @@
...
-
-## Platform Version 0.8.1 - 
-* Make Fluvio Client compatitble with WASM([#1042](https://github.com/infinyon/fluvio/pull/1042))
-* Improve replication logic([#1011](https://github.com/infinyon/fluvio/pull/1011))
-
+## Platform Version 0.8.1 - 2020-05-03
+* Add GH Release step to release, update changelog ([#1024](https://github.com/infinyon/fluvio/pull/1024))
+* Fix Docker image publish flow ([#1025](https://github.com/infinyon/fluvio/pull/1025))
+* Use file name for the external commands (fixes #889) ([#1008](https://github.com/infinyon/fluvio/pull/1008))
+* Disable test that is flaky on CI ([#1018](https://github.com/infinyon/fluvio/pull/1018))
```
Comment (sehz) [inline CHANGELOG.md:6]: don't need to include this as well

**Review by sehz — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,9 +1,13 @@
...
-
+## Platform Version 0.8.1 - 2020-05-03
+* Add GH Release step to release, update changelog ([#1024](https://github.com/infinyon/fluvio/pull/1024))
+* Fix Docker image publish flow ([#1025](https://github.com/infinyon/fluvio/pull/1025))
+* Use file name for the external commands (fixes #889) ([#1008](https://github.com/infinyon/fluvio/pull/1008))
+* Fix Fluvio log directory on K8 ([#1043](https://github.com/infinyon/fluvio/pull/1043))
+* Add RecordKey API for sending records without keys ([#985](https://github.com/infinyon/fluvio/pull/985))
+* Make Fluvio Client compatitble with WASM ([#1042](https://github.com/infinyon/fluvio/pull/1042))
+* Update Replication logic for SPU ([#1011](https://github.com/infinyon/fluvio/pull/1011))
+* Bump up fluvio-protocol ([#1049](https://github.com/infinyon/fluvio/pull/1049))
```
Comment (sehz) [inline CHANGELOG.md:10]: no need for 1049

File: src/client/Cargo.toml [hunk 4]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "fluvio"
-version = "0.8.0"
+version = "0.8.1"
```
Comment (sehz) [inline src/client/Cargo.toml:3]: why 0.8.1 ?

Comment (tjtelan) [reply src/client/Cargo.toml:3]: I must have done that unconsciously. I will need to yank that crate

**Review by sehz — APPROVED** [review]

**Pull request merged**